.github/workflows/action-conda-publish.yml

```
@@ -0,0 +1,85 @@
+name: Anaconda Publish
+
+on:
+  release:
+    types: [published]
+  # use this to test before actual release and publish
+  push:
+    branches:
+      - master
+
+jobs:
+
+  publish:
```

Consider using slightly more descriptive job names

```suggestion
  deploy_to_conda:
```

---

cheers @stefsmeets - no need to since the job appears as `Anaconda Publish/publish` - ideally we'd want shorter names so they don't get truncated in the browser dashboard